install_config/cluster_metrics.adoc

```diff
@@ -306,15 +340,15 @@ ifdef::openshift-enterprise[]
 ====
 endif::[]
 
-
-You will need to save your completed file with the file name *_metrics.yaml_*.
+Copy this file to another directory with the file name
+*_metrics-deployer.yaml_*.
```

Comment:
@mwringe I was trying to consider both people that would just use the vanilla one as is as well as folks that might be modifying the template. My thinking was having everyone just copy it someplace else would leave the vanilla one in tact, and also it made the `oc process` cmd a little cleaner later. I agree this is not the cleanest solution.

If you think it's fine to just have people editing the original in place and `oc create` from that location, I can update it.